examples/2D_kelvin_helmholtz/case.py

```
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
```

Ruff (0.14.5)

Shebang is present but file is not executable

(EXE001)